addons/mrp/report/mrp_production_templates.xml

```diff
@@ -41,18 +45,24 @@
                             <span t-field="o.product_qty"/>
                             <span t-field="o.product_uom_id.name" groups="uom.group_uom"/>
                         </div>
+                         <div class="col-3" t-if="o.qty_producing">
+                            <strong>Quantity Producing:</strong><br/>
+                            <span t-field="o.qty_producing"/> Units
```

Comment:
You should add another `<span t-field="o.product_uom_id.name" groups="uom.group_uom"/>` here instead of hardcoding in `Units` since the quantity producing may not be in units.